services/theme_update.py

```diff
@@ -222,7 +222,7 @@ def application(environ, start_response):
                 return ['']
             start_response(status, update.get_headers(len(output)))
         except:
-            log_exception(data)
```

Comment:
In this case data is `data = environ['wsgi.input'].read()` which is actually an empty body. You'll see in the logs it shows `Data:  :...` and then goes on to give the traceback. So `data` is pretty damn useless at the moment.